src/components/DocumentationTopic/Title.vue

```diff
@@ -35,14 +35,14 @@ export default {
 
 .eyebrow {
   @include font-styles(eyebrow-reduced);
-  color: light-color(fill-gray-tertiary);
+  color: var(--hero-eyebrow);
```

Comment:
Minor thing but maybe `--hero-eyebrow-color` would be better indicate what the property is used for?